selenium_test.py

```diff
@@ -0,0 +1,209 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select
+import pandas as pd
+
+KEYWORDS_FILE = 'textfile.txt'
+RANKED_OUTPUT_FILE = 'Output.csv'
+
+
+title_list = []
+description_list = []
+category_list = []
+keyword_list = []
+link_list = []
+
+def read_keywords():
+    with open(KEYWORDS_FILE) as fd:
+        return [keyword.strip() for keyword in fd.readlines()]
+
+
+
+
+def selenium_login():
```

Comment:
1. This function should have a better name

2. It should take in a single keyword, as well as start and end years, and just do the search for that.

3. The lists of results (`title_list`, etc.) should be local to this function.